src/bucket/BucketManagerImpl.cpp

```diff
@@ -670,10 +695,10 @@ BucketManagerImpl::forgetUnreferencedBuckets()
         // have references to them, just us. It's ok to retain a few too
         // many buckets, a little longer than necessary.
         //
-        // This conservatism is important because we want to enforce that only
-        // one bucket ever exists in memory with a given filename, and that
-        // we're the first and last to know about it. Otherwise buckets might
-        // race on deleting the underlying file from one another.
+        // This conservatism is important because we want to enforce that
```

Comment:
I thought you mentioned before that indexes for buckets that are only referenced by the publish queue were using a lot of memory - this seems to still be the case with this PR?